src/jackson.js

```diff
@@ -17,31 +15,41 @@ app.use(express.urlencoded({ extended: true }));
 
 app.get(oauthPath + '/authorize', async (req, res) => {
   try {
-    await oauthController.authorize(req, res);
+    const { redirect_url } = await oauthController.authorize(req.query);
+
+    res.redirect(redirect_url);
   } catch (err) {
     res.status(500).send(err.message);
   }
 });
 
 app.post(env.samlPath, async (req, res) => {
   try {
-    await oauthController.samlResponse(req, res);
+    const { redirect_url } = await oauthController.samlResponse(req.body);
+
+    res.redirect(redirect_url);
   } catch (err) {
     res.status(500).send(err.message);
```

Comment:
statusCode from JacksonError should be used here

Reply:
Are you talking about using the status code in the statement `res.status(500).send(err.message);` right?

Reply:
@devkiran exactly

Reply:
Ok, noted.